lib/util/google_sign_in.dart

```diff
@@ -34,12 +41,10 @@ class GoogleSignMeIn {
     return ' ';
   }
 
-  logout() async {
+  Future<User> logout() async {
```

Comment:
The function has a return type of `Future<User>` but it's not returning anything.
Change the return type to `Future<void>`